pom.xml

```diff
@@ -256,7 +256,17 @@
                 </includes>
                 <filtering>true</filtering>
             </resource>
+            <resource>
+                <directory>src/main/resources/</directory>
```

Comment:
At the moment, not at all since I've only just created it. I've mostly only ever seen the resource directory targeted in general. It might become an issue if the resources gets filled up with many files, but that's when you end up with some system like language files or whatever bundling there is.